src/TestSuite/IntegrationTestTrait.php

```diff
@@ -282,6 +282,9 @@ public function enableSecurityToken($unlockedFields = [])
 
     /**
      * Set list of fields that are excluded from field validation.
+     *
+     * @param array $unlockedFields List of fields that are excluded from field validation.
```

Comment:
Isnt this `string[]` ?

Reply:
I copied the comment from SecureFieldTokenTrait but you are right. Maybe fix it in both files.

Reply:
I will take care of it in 4.x: https://github.com/cakephp/cakephp/pull/13485
but you are welcome to fix in 3.x